src/libraries/System.Text.Json/tests/Utf8JsonReaderTests.MultiSegment.cs

```diff
@@ -486,6 +486,12 @@ public static void TestJsonReaderUtf8SegmentSizeOne(bool compactData, TestCaseTy
         [MemberData(nameof(LargeTestCases))]
         public static void TestJsonReaderLargeUtf8SegmentSizeOne(bool compactData, TestCaseType type, string jsonString)
         {
+            // Skipping really large JSON on Browser to prevent OOM
+            if (PlatformDetection.IsBrowser && (type == TestCaseType.Json40KB || type == TestCaseType.Json400KB || type == TestCaseType.ProjectLockJson))
```

Comment:
nit: use switch

Reply:
There's existing code that uses the same pattern so I think it's better to be consistent.